pattern-library/sass/patterns/_icons.scss

```diff
@@ -497,3 +482,9 @@ cc;
     background: url('#{$edx-icons-font-path}/fallback-img/#{$edx-icon}.svg') center center no-repeat;
   }
 }
+
+@each $synonym, $edx-icon in $edx-icon-synonyms {
+    .icon-fallback-img .icon-#{$synonym} {
+    background: url('#{$edx-icons-font-path}/fallback-img/#{$edx-icon}.svg') center center no-repeat;
```

Comment:
It might be nice to pull this line and [the similar one above ~482](https://github.com/edx/ux-pattern-library/pull/284/files#diff-edfc1e00417629e4a2621e84f5676c68R482) into a mixin so that if one changes for some reason the other will follow suit.